src/utils.rs

```diff
@@ -0,0 +1,11 @@
+use base64::Engine;
+use base64::engine::general_purpose;
+use sha2::Digest;
+
+pub fn create_base64_encoded_hash(data: String) -> String {
```

Comment:
Let's merge it with SDJWTCommon functions. We can keep SDJWTCommon method as is and move the functions to src/utils

Reply:
Done, pls check